appengine/flexible/WebSocket/app.yaml

```diff
@@ -0,0 +1,13 @@
+runtime: aspnetcore
```

Comment:
Can we additionally include the commented-out session affinity line and associated comment documentation present in the Python sample? https://github.com/GoogleCloudPlatform/python-docs-samples/blob/db9218fa0a186db254cedc1fa5eb0b4980d69737/appengine/flexible/websockets/app.yaml#L17

Reply:
Fixed.